source/releasenotes/about.rst

```diff
@@ -118,3 +118,17 @@ After identifying the snapshots with a backing store and the related templates,
 
 .. parsed-literal::
    qemu-img convert -O qcow2 -U --image-opts driver=qcow2,file.filename=<path to snapshot on secondary storage> <path to snapshot on secondary storage>-converted
+
+Issue regarding LDAP authentication on version 4.19.0
+=====================================================
+
+In version 4.19.0, the encryption of scoped configurations of Accounts and Domains was changed to only encrypt if there were sensitive data (e.g, they belonged to the Hidden or Secure category) as the all configurations for Accounts and Domains were encrypted. However, when using the encrypted values from these scopes, ACS did not correctly decrypt these values. For this reason, a simple solution was to update these configurations to their plain values with manual DB intervention, as reported in issue `#8637`.
+
+For users that manually set the configurations ``ldap.bind.password`` and ``ldap.truststore.password`` to a plain value in order to fix the faulty behaviour, it is required to store them encrypted. It will not be possible to update the configuration via UI, as an exception will be thrown when ACS tries to decrypt the plain value. To fix this, it is required to set the password again for ACS to encrypt it. There are two options:
```

Comment:
@BryanMLima 
maybe add more word ?
`The issue has been fixed in CloudStack 4.19.1.0. For users that manually set the configurations ``ldap.bind.password`` and ``ldap.truststore.password`` to a plain value in order to fix the faulty behaviour, it is required to store them encrypted after upgrading to 4.19.1 and onwards`.

Reply:
@weizhouapache, done, just made a few changes.